components/infobox/wikis/brawlhalla/infobox_league_custom.lua

```diff
@@ -0,0 +1,189 @@
+	['1v1'] = '1v1',
+	['2v2'] = '2v2',
+	['3v3'] = '3v3',
+}
+_MODES.solo = _MODES['1v1']
+_MODES.singles = _MODES['1v1']
+_MODES.duo = _MODES['2v2']
+_MODES.doubles = _MODES['2v2']
+_MODES.default = _MODES['1v1']
+
+local _league
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_league = league
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _league.args
+	table.insert(widgets, Cell{
+		name = 'Mode',
+		content = {CustomLeague:_displayMode(args.mode)}
+	})
+	table.insert(widgets, Cell{
+		name = 'Misc Mode:',
+		content = {args.miscmode}
+	})
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	local args = _league.args
+	if id == 'customcontent' then
+		if not String.isEmpty(args.team_number) then
+			table.insert(widgets, Title{name = 'Teams'})
+			table.insert(widgets, Cell{
+				name = 'Number of teams',
+				content = {args.team_number}
+			})
+		end
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool(args)}
+			},
+		}
+	elseif id == 'liquipediatier' then
+		return {
+			Cell{
+				name = 'Liquipedia Tier',
+				content = {CustomLeague:_createLiquipediaTierDisplay(args)}
+			},
+		}
+	end
+	return widgets
+end
+
+function CustomLeague:_createLiquipediaTierDisplay(args)
+	local tier = args.liquipediatier or ''
+	local tierType = args.liquipediatiertype or ''
+
+	if String.isEmpty(tier) then
+		return nil
+	end
+
+	local function buildTierString(tierString)
+		local tierText = Tier.text[tierString]
+		if not tierText then
+			table.insert(_league.warnings, tierString .. ' is not a known Liquipedia Tier/Tiertype')
+			return ''
+		else
+			return '[[' .. tierText .. ' Tournaments|' .. tierText .. ']]'
+		end
+	end
+	local tierDisplay = buildTierString(tier)
+	if String.isNotEmpty(tierType) then
+		tierDisplay = buildTierString(tierType) .. '&nbsp;(' .. tierDisplay .. ')'
+	end
+
+	return tierDisplay
+end
+
+function CustomLeague:_createPrizepool(args)
+	if String.isEmpty(args.prizepool) and String.isEmpty(args.prizepoolusd) then
+		return nil
+	end
+
+	local date
+	if String.isNotEmpty(args.currency_rate) then
+		date = args.currency_date
+	end
+
+	return PrizePoolCurrency._get({
+		prizepool = args.prizepool,
+		prizepoolusd = args.prizepoolusd,
+		currency = args.localcurrency,
+		rate = args.currency_rate,
+		date = date or Variables.varDefault('tournament_enddate', _TODAY),
+	})
+end
+
+function CustomLeague:_displayMode(mode)
+	if String.isEmpty(mode) then
+		return _MODES.default
+	end
+
+	return _MODES[mode:lower()] or _MODES.default 
+end
+
+function CustomLeague:defineCustomPageVariables(args)
```

Comment:
probably needs several additional legacy vars:
- `tournament_valve_premier`
- `tournament_pro_circuit_tier`
- `tournament_gamemode`
- `tournament_tier`(standardized variable name is `tournament_liquipediatier`, but until that has been adjusted everywhere ...)